src/index.html

```diff
@@ -1,13 +1,42 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/style.css">
+  </head>
+  <body>
+    <div class="card page__card" data-qa="card">
+      <img 
+        src="./images/imac.jpeg" 
+        alt="computerImage" 
+        class="card__computerImage"
+      >
+      <p class="card__computerName">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+      <p class="card__computerCode">Код товара: 195434</p>
+      <div class="rating">
+        <div class="rating__stars-4">
```

Comment:
```suggestion
        <div class="rating__stars stars stars--4">
```
- Класс rating__stars устанавливает внешние отступы для блока stars
- Класс stars определяет внутреннюю разметку блока
- Класс stars--4 - модификатор, определяющий количество активных звездочек